pilot/pkg/bootstrap/namespacecontroller.go

```diff
@@ -116,6 +125,31 @@ func NewNamespaceController(data func() map[string]string, kubeClient kubernetes
 				return c.namespaceChange(obj)
 			})
 		},
+		UpdateFunc: func(newObj, oldObj interface{}) {
```

Comment:
Why do we need to handle updates? it seems to me like we don't really care about a NS changing, we just need to handle it being created. Is there some client-go thing where this matters or a case I am missing?

I know this will be called every resyncPeriod but we have resync disabled currently